tests/useMeasure.test.ts

```diff
@@ -175,3 +180,19 @@ it('calls .disconnect() on ResizeObserver when component unmounts', () => {
 
   expect(disconnect).toHaveBeenCalledTimes(1);
 });
+
+it('ref accepts null value', () => {
+  let observe = jest.fn();
+  (window as any).ResizeObserver = class ResizeObserver {
+    observe = observe;
+  };
+
+  const { result } = renderHook(() => useMeasure());
+  const [ref] = result.current;
+
+  act(() => {
+    ref(null);
+  });
+
+  expect(observe).not.toHaveBeenCalled();
+})
```

Comment:
New test to make sure `ref` accepts `null` and that `useMeasure` doesn't try to observe a `null` element.